core/src/main/java/de/danoeh/antennapod/core/util/syndication/HtmlToPlainText.java

```diff
@@ -26,8 +37,41 @@
  */
 public class HtmlToPlainText {
 
+    /**
+     * Use this method to strip off HTML encoding from given text
+     * <p>
+     * Replaces bullet points with *, ignores colors/bold/...
+     *
+     * @param str String with any encoding
+     * @return Human readable text with minimal HTML formatting
+     */
+    public static Spanned getPlainText(@NotNull Spanned str) {
```

Comment:
Why aren't return type and parameters simple strings? This way, you have to call toString whenever using the parameter.